trivup/clusters/KafkaCluster.py

```diff
@@ -250,7 +250,7 @@ def _setup_client_conf(self):
                     break
 
             elif self.sasl_mechanism == 'OAUTHBEARER':
-                if self.oidc is not None:
+                if hasattr(self, 'oidc'):
```

Comment:
I think we want to set `self.oidc = None` in the constructor instead, like we do with the other stuff (.ssl, .zk, etc)

Reply:
Updated now